homeassistant/components/fully_kiosk/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Account is already configured"
+        },
+        "error": {
+            "cannot_connect": "Failed to connect",
+            "unknown": "Unexpected error"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "host": "Host",
+                    "password": "Password",
+                    "ssl": "Uses an SSL certificate",
+                    "verify_ssl": "Verify SSL certificate"
+                }
+            }
+        }
+    }
+}
```

Comment:
This file is no longer included in the git repository, please remove it

Reply:
Without it the integration setup dialog has no text and the generic error messages lose the user-friendly verbiage.

![image](https://user-images.githubusercontent.com/19595786/225502890-0b60a354-9507-471c-ab8c-624d7bbb944c.png)

Reply:
You need to run `python3 -m script.translations develop --all` in your local testing.
Again, please remove the file from this PR.

You may also be basing your work on an old dev branch; the translations were removed from the core repo a couple of months ago.

Reply:
Got it. Removed. `script.translations` generated the en.json file locally.